library/arith.lisp

```diff
@@ -119,8 +131,20 @@ The fields are defined as follows:
     ;; integer greater or equal to X, and the greatest integer less
     ;; than or equal to X, along with their respective remainders
     ;; expressed as values of type :T.
-    (quantize (:a -> (Quantization :a)))))
+    (quantize (:a -> (Quantization :a))))
 
+  ;;
+  ;; Rational
+  ;;
+
+  (define-class ((Quantizable :a) (Into :a Fraction) => Rational :a)
+    "Any real number that can be exactly represented by a fraction of integers."
+    (numerator (:a -> Integer))
+    (denominator (:a -> Integer)))
+
+  (declare rational->fraction (Rational :a => :a -> Fraction))
```

Comment:
Is this necessary?

Reply:
No, it just seemed like a common enough operation (especially when it was `toFraction`)
Now that I take a closer look, it only saves a single space:

`(the Fraction (into _))`
`(rational->fraction _)`

I will remove it.